index.js

```diff
@@ -11,13 +11,40 @@ const MONGODB_URI = 'mongodb://localhost:27017/recipe-app';
 mongoose
   .connect(MONGODB_URI)
   .then(x => {
-    console.log(`Connected to the database: "${x.connection.name}"`);
+    //console.log(`Connected to the database: "${x.connection.name}"`);
     // Before adding any recipes to the database, let's remove all existing ones
     return Recipe.deleteMany()
   })
   .then(() => {
-    // Run your code here, after you have insured that the connection was made
+    // iteration 2
+    // Recipe.create(data[0]);
+
+    return Recipe.insertMany(data);
+  })
+  .then( result => {
+    return Recipe.find({duration :{$gte:0}} ,"title" );
+  })
```

Comment:
here you dont need to use find() method,,< you can get the result (this is an array) and you can iterate this array with forEach or every loop you want and print every title of element of result array :
` .then(() => {
    return Recipe.insertMany(data)
  })
  .then((recipesArr) => {
    recipesArr.forEach((recipe) => {
      console.log(recipe.title);
    })`